channels/whatsapp/index.rst

```diff
@@ -0,0 +1,247 @@
+   the customer. If the customer isn't replying, the conversation ends
+   after 24 hours. You can't communicate with this customer via WhatsApp due to
+   their policy until a new conversation is started by your customer.
+
+   If the ticket can not be closed and to keep the conversation ongoing, we
+   suggest to create a dedicated trigger for the WhatsApp channel. For example
+   you could send a auto-generated message to the customer after 20 hours and
+   ask to reply to your message to keep the conversation open.
+
+Customer initiated communication only
+   The WhatsApp channel only supports customer initiated communication.
+   For business initiated communication, it would be necessary to create a
+   message template which has to be verified by Meta first.
+
+Supported media types
+   Currently, only text messages, documents and videos are supported.
+
+One attachment per article
+   Currently, it's only possible to attach one file per outgoing message.
+
+File size for attachments
+   There are different maximum file sizes for different media types. To pick
+   the most important ones:
+
+   - audio: 16 MB
+   - document: 100 MB
+   - image: 50MB
+
+   To read more in detail, have a look on their `documentation of supported
+   media types <https://developers.facebook.com/docs/whatsapp/cloud-api/reference/media?locale=en_US#supported-media-types>`_.
+
+One phone number per app
+   If you want to use more than one phone number, you have to create a business
+   app for each number you want to use and add it as a separate channel in
+   Zammad.
+
+Setup
+-----
+
+First of all, make sure to match the prerequisites and to read the limitations
+above. If you did not setup your Meta developer account, Meta business account
+and your developer app, first go to our :doc:`preparation page <preparation>`
+and come back after finishing it.
+
+Then head over to Zammad's admin panel to add the WhatsApp channel under
+"Channels > WhatsApp > Add Account" and follow the steps which are described
+below.
+
+Step 1
+^^^^^^
+
+In the first step you have to provide credentials of your account/app:
+
+.. figure:: /images/channels/whatsapp/setup-step-1.png
+   :alt: Step 1 of adding a WhatsApp account
+   :align: center
+   :scale: 80%
+
+WhatsApp Business Account ID
+   Enter your WhatsApp Business Account ID. You can find it in your app
+   under "WhatsApp > API Setup":
+
+   .. figure:: /images/channels/whatsapp/business-account-id.png
+     :alt: App dashboard with business account ID
+     :align: center
+
+Access Token
+   Enter the access token from your app. You can create it in the
+   `business settings <https://business.facebook.com/settings/system-users>`_
+   under "System Users". Be aware that you have to copy the token, it is just
+   displayed once!
+
+   .. figure:: /images/channels/whatsapp/created-system-user.png
+      :alt: Get your app token from the system user
+      :align: center
+
```

Comment:
Perhaps add here a requirement that the system user has to have `Admin access`, as employee would not work with `whatsapp_business_management` permission. If the user is missing, it needs to be created!